### DISCUSSION TITLE
解析xml2bean，带前缀的没有例子，请教一下

### DISCUSSION THREAD

**Discussion by lunasaw** [looly/hutool#3044] [Q&A]
![image](https://user-images.githubusercontent.com/52351139/229997057-29a11911-1792-4c98-9dff-b807600f1948.png)
要解析这种前缀的 xml2bean 怎么注解呀
cn.hutool.core.util.JAXBUtilTest#xmlToBeanTest 这个好像没例子

**Comment by looly** [comment]
这个好像不行。你需要定义xmlns

**Reply by lunasaw** [reply]
手动注解xmlns，能现有方法能实现解析吗
